plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcModule.java

```diff
@@ -44,12 +45,32 @@
 public class JdbcModule
         extends AbstractConfigurationAwareModule
 {
+    private static class RetryingConnectionFactoryProvider
```

Comment:
What is the goal of this provider?

Reply:
That was the 'previous' approach before we changed the acceptance criteria. This code is now deleted.